src/index.ts

```diff
@@ -21,7 +22,10 @@ async function main() {
     let path: string[] = options.file.replace('.json', '').split('/')
     let programName: string = path[path.length - 1]
     const paths = new Paths(`./`, programName)
-    const idl = JSON.parse(readFileSync(paths.idlFile(programName), "utf8"))
+    const idl: Idl = JSON.parse(readFileSync(paths.idlFile(programName), "utf8"))
+    idl.metadata = {
+      address: "PROGRAM PUBKEY"
```

Comment:
won't this overwrite the address, in case that it is actually provided in the JSON?